pom.xml

```diff
@@ -377,9 +377,17 @@
                     <exclude>org.sl4fj:slf4j-jcl</exclude>
                     <exclude>log4j:log4j</exclude>
                     <exclude>ch.qos.logback:logback-classic</exclude>
+                    <!-- NOTE: We're banning any HBase deps versions other than the approved ${hbase.version},
+                               which is aimed at preventing the classpath collisions w/ transitive deps usually) -->
+                    <exclude>org.apache.hbase:hbase-common:*</exclude>
+                    <exclude>org.apache.hbase:hbase-client:*</exclude>
+                    <exclude>org.apache.hbase:hbase-server:*</exclude>
                   </excludes>
                   <includes>
                     <include>org.slf4j:slf4j-simple:*:*:test</include>
+                    <exclude>org.apache.hbase:hbase-common:${hbase.version}</exclude>
+                    <exclude>org.apache.hbase:hbase-client:${hbase.version}</exclude>
+                    <exclude>org.apache.hbase:hbase-server:${hbase.version}</exclude>
                   </includes>
```

Comment:
why are these "exclude"s repeated here under "includes" ?

Reply:
Good catch! I'm not actually sure how Maven is working w/ these though (i checked enforcer and it worked)